src/content/docs/zh/developers/l1-and-l2-bridging/enforced-transactions.mdx

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
translate or not?

Reply:
Same as "estimating-gas-and-tx-fees.mdx", I didn't see this page show up in local development and online docs in English. And I check the Spanish version, they also remains untransalted. 
I guess it's a page working in progress?